Makefile

```diff
@@ -20,9 +20,17 @@ test: generate
 	echo 'mode: atomic' > coverage.txt && go test -covermode=atomic -coverprofile=coverage.txt -v -race -timeout=30s ./...
 
 # Run the integration tests. Requires github token for scorecard (GITHUB_AUTH_TOKEN=<your token>)
+# To run it locally you can run the following command: make start-integration-service
 .PHONY: integration-test
 integration-test: generate check-env
-	go test -tags=integration ./...
+	 go test -tags=integration ./...
+
+# Runs the integration tests locally using docker-compose to start the dependencies and cleans up after itself.
+.PHONY: integration-test-local
+integration-test-local: generate check-env start-integration-service
+	sleep 5
```

Comment:
can we replace this to a check if the service is up instead? like a HTTP query to the GQL endpoint to ensure it's up?

Reply:
Trying some bash foo with help of LLM ;)